fe/app/model/Page.ts

```diff
@@ -0,0 +1,6 @@
+export type Page =
```

Comment:
Shouldn't we have also in here the home page?

Reply:
Yes agree